test/spec/Feature/Query/RpcSpec.hs

```diff
@@ -1274,9 +1274,9 @@ spec actualPgVersion =
             `shouldRespondWith`
               [json|{
                 "hint": "Perhaps you meant to call the function test.unnamed_text_param",
-                "message": "Could not find the test.unnamed_int_param(x, y) function or the test.unnamed_int_param function with a single unnamed json or jsonb parameter in the schema cache",
+                "message": "Could not find the function test.unnamed_int_param(x, y) in the schema cache",
                 "code":"PGRST202",
-                "details":null
+                "details":"Searched for the function test.unnamed_int_param(x, y) or with a single unnamed json/jsonb parameter, but no matches were found in the schema cache."
```

Comment:
Hm, this is not completely clear.

Maybe we should decompress the "(x,y)" part, this would also clarify what we mean on the message. Otherwise it seems redundant.

Reply:
```suggestion
                "details":"Searched for the function test.unnamed_int_param with x, y parameters or with a single unnamed json/jsonb parameter, but no matches were found in the schema cache."
```

Reply:
Maybe something like this?

```suggestion
                "details":"Searched for the function test.unnamed_int_param with parameters x, y or with a single unnamed json/jsonb parameter, but no matches were found in the schema cache."
```

Reply:
Yes, that would be better.